### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Message Window_ScreenReader: Screen reader JAWS is not announcing table elements under Messages using Table keys (Ctrl + Alt + Arrow keys)

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17621]
**Actual:**
Screen reader JAWS is not announcing table elements under Messages using Table keys (Ctrl + Alt + Arrow keys)

**Note:**

1. Issue is also repro for results table
2. Issue is no repro with NVDA Screen reader.

**Expected:**
Screen Reader JAWS should announce elements under Messages dropdown with proper information including column header. 

GitHubTag:#A11yTCS; #Win11; #DesktopApp; #BM_mssql for VSCode_Win32_Mar2023; #A11ySev2; #Benchmark; #mssql for VSCode; #ScreenReader; #JAWS; #FTP; #WCAG1.3.1;

**Environment Details:**
Application Name: mssql for VSCode
Operating System: Win11[22H2]
Visual studio code insider: 1.77.0-insider (user setup)
Extension Version: v1.17.1

**Repro Steps:**

1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter
10. Tab till Execute Query and hit enter
11. Tab till "Messgaes" dropdown and hit enter
12. Turn on the screen reader JAWS
13.Tab till Table under "Messages" and use Table keys (Ctrl + Alt + Arrow keys)

**User Impact:**
Screen reader users will not able to navigate through the elements of table under as there is no other way for them get the info under table.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Attachment**

**Comment by Roopesh-T** [comment]
Issue not repro in Version: 1.80.0-insider (user setup)
![17621_Closed](https://github.com/microsoft/vscode-mssql/assets/86601127/b7c08351-3077-4283-ba94-dc06e54f7321)
**GitTags**: #closed;#regressed:06-22-23;
